analyzer/runtime/extract.go

```diff
@@ -727,6 +728,144 @@ func extractEvents(blockData *BlockData, relatedAccountAddresses map[apiTypes.Ad
 					}
 					return nil
 				},
+				ERC721Transfer: func(fromEthAddr []byte, toEthAddr []byte, tokenIDU256 []byte) error {
+					tokenID := &big.Int{}
+					tokenID.SetBytes(tokenIDU256)
+					fromZero := bytes.Equal(fromEthAddr, uncategorized.ZeroEthAddr)
+					toZero := bytes.Equal(toEthAddr, uncategorized.ZeroEthAddr)
+					if !fromZero {
+						fromAddr, err2 := registerRelatedEthAddress(blockData.AddressPreimages, relatedAccountAddresses, fromEthAddr)
+						if err2 != nil {
+							return fmt.Errorf("from: %w", err2)
+						}
+						eventData.RelatedAddresses[fromAddr] = true
+					}
+					if !toZero {
+						toAddr, err2 := registerRelatedEthAddress(blockData.AddressPreimages, relatedAccountAddresses, toEthAddr)
+						if err2 != nil {
+							return fmt.Errorf("to: %w", err2)
+						}
+						eventData.RelatedAddresses[toAddr] = true
+					}
+					// todo: reckon ownership
```

Comment:
this PR doesn't add token ownership tracking. there are no 'balances' in this current state of ERC-721 support

Reply:
gonna all-caps the todo. marking as resolved